akka-docs/_sphinx/themes/akka/static/warnOldDocs.js

```diff
@@ -0,0 +1,70 @@
+jQuery(document).ready(function ($) {
+
+  var latestInSeries = {
+    "2.2": {
+      latest: "2.2.5",
+      deprecated: true,
+      instead: "2.3"
+    },
+    "2.3": {
+      latest: "2.3.13"
+    },
+    "2.4": {
+      latest: "2.4.0-RC1"
+    },
+
+    // -- RP --
+    "rp-15v01": {
+      rp: true,
+      latest: "rp-15v01p05"
+    }
+  };
+
+  //if (location.hostname === "doc.akka.io") {
+  for (var series in latestInSeries) {
+    if (browsedVersion().startsWith(series)) {
+      floatyWarning(browsedVersion(), latestInSeries[series])
+    }
+  }
+  //}
+
+  function floatyWarning(version, seriesInfo) {
+    var targetUrl = ("" + window.location).replace(version, seriesInfo.latest);
+    var $floatyWarning = $('<div id="floaty-warning"/>');
+
+    $floatyWarning
+        .html('' +
+        'You are browsing the docs for Akka [' + version + '], ' +
+        'however the latest release in this series is: ' +
+        '<strong><a href="' + targetUrl + '">[' + seriesInfo.latest + ']</a></strong>. <br/>' +
+        '<a href="' + targetUrl + '">' +
+        '<string>Click here to go to the latest [' + seriesInfo.latest + '] version of the docs.</strong>' +
+        '</a>');
+
+    if (!!seriesInfo['deprecated']) { // yes !!, because: undefined => false, false => false, true => true
+      var instead = seriesInfo['instead'];
+      var unsupportedMessage = '<p style="font-weight: bold">' +
+          'This version of Akka (' + version + ') has been end-of-lifed and is currently unsupported! ' +
+          'Please upgrade to ' + instead + ' (or later) as soon as possible.' +
+          '</p>';
+      $floatyWarning.prepend(unsupportedMessage)
+    }
+
+    $floatyWarning
+        .hide()
+        .prependTo("body")
+        .fadeIn()
+        .delay(10 * 1000)
+        .fadeOut()
+  }
+
+  // e.g. "docs/akka/2.3.10/scala/persistence.html" => "2.3.10"
+  function browsedVersion() {
+    var path = window.location.pathname;
+    var path2 = path.substring(path.indexOf("2")); // TODO quite hacky
```

Comment:
Isn't the probability that we'd release akka 3 at some time pretty high?

Match with regexp instead? 
```
var rx = /\/(\d[^/]*)/
var match = rx.exec(path)
if (match && match[1]) return match[1]
else errorOrSomething 
```